app/controllers/validations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ValidationsController < ApplicationController
+  before_action :assign_params
+
+  def show
+    @user ||= User.new(user_params)
+    @user.validate
+  end
+
+  private
+
+  def assign_params
+    self.params = @reflex_params if @reflex_params.present?
```

Comment:
I do think that this is quite clever, but I would suggest it start life as a [pattern](https://docs.stimulusreflex.com/patterns#server-side) we recommend as opposed to something in core.

Reply:
Agree that it's too premature, but I think we'll find that this pattern becomes pervasive enough to justify adding to core. Time will tell.

Reply:
If it does become core, I would propose adding a params accessor to reflex.rb and then merging it into the controller params. Seems a little more idiomatic? :man_shrugging: